app/models/piece.rb

```diff
@@ -2,6 +2,11 @@ class Piece < ActiveRecord::Base
   belongs_to :user
   belongs_to :game
 
+  def move_piece(new_x, new_y)
```

Comment:
If move_piece method is method you can run on a piece, lets give it a name of move_to! for instance. This way your call looks like:

@piece.move_to!(new_x, new_y)  

and is very readable.